src/crypto/random-browser.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+const Buffer = require('safe-buffer').Buffer
+
+function getWebCrypto () {
+  if (typeof window !== 'undefined') {
```

Comment:
Requiring libp2p-crypto seems fine.  I doubt many people will want to use this module without it anyway.